library/src/main/java/com/novoda/downloadmanager/lib/DownloadBatch.java

```diff
@@ -0,0 +1,38 @@
+package com.novoda.downloadmanager.lib;
+
+import android.content.ContentValues;
+
+import java.util.List;
+
+public class DownloadBatch {
```

Comment:
We want clients to create this right now, the next PR will have a better API than this